diff --git a/src/scikit_build_core/build/_wheelfile.py b/src/scikit_build_core/build/_wheelfile.py
index 3b3740b..f393c31 100644
--- a/src/scikit_build_core/build/_wheelfile.py
+++ b/src/scikit_build_core/build/_wheelfile.py
@@ -25,7 +25,7 @@ if TYPE_CHECKING:
     from packaging.tags import Tag
 
     from .._compat.typing import Self
-    from .._vendor.pyproject_metadata import StandardMetadata
+    from pyproject_metadata import StandardMetadata
 
 EMAIL_POLICY = EmailPolicy(max_line_length=0, mangle_from_=False, utf8=True)
 
diff --git a/src/scikit_build_core/build/generate.py b/src/scikit_build_core/build/generate.py
index 03547f3..df551a7 100644
--- a/src/scikit_build_core/build/generate.py
+++ b/src/scikit_build_core/build/generate.py
@@ -7,7 +7,7 @@ import string
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from .._vendor.pyproject_metadata import StandardMetadata
+    from pyproject_metadata import StandardMetadata
     from ..settings.skbuild_model import GenerateSettings
 
 
diff --git a/src/scikit_build_core/build/metadata.py b/src/scikit_build_core/build/metadata.py
index 68c1082..7bedd91 100644
--- a/src/scikit_build_core/build/metadata.py
+++ b/src/scikit_build_core/build/metadata.py
@@ -4,7 +4,7 @@ from typing import TYPE_CHECKING, Any
 
 from packaging.version import Version
 
-from .._vendor.pyproject_metadata import StandardMetadata
+from pyproject_metadata import StandardMetadata
 from ..settings._load_provider import load_dynamic_metadata
 
 if TYPE_CHECKING:
diff --git a/tests/test_wheelfile_utils.py b/tests/test_wheelfile_utils.py
index 9e92ac5..747d482 100644
--- a/tests/test_wheelfile_utils.py
+++ b/tests/test_wheelfile_utils.py
@@ -4,7 +4,7 @@ import zipfile
 from packaging.tags import Tag
 
 import scikit_build_core.build._wheelfile
-from scikit_build_core._vendor.pyproject_metadata import StandardMetadata
+from pyproject_metadata import StandardMetadata
 
 
 def test_wheel_metadata() -> None:
